test/forge/invariant/MorphoDynamicInvariantTest.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import "./BaseMorphoInvariantTest.sol";
+
+contract MorphoDynamicInvariantTest is BaseMorphoInvariantTest {
+    using MorphoLib for IMorpho;
+    using MarketParamsLib for MarketParams;
+
+    uint256 internal immutable MIN_PRICE = ORACLE_PRICE_SCALE / 10;
+    uint256 internal immutable MAX_PRICE = ORACLE_PRICE_SCALE * 10;
+
+    function setUp() public virtual override {
+        _weightSelector(this.liquidateSeizedAssetsNoRevert.selector, 2);
+        _weightSelector(this.liquidateRepaidSharesNoRevert.selector, 2);
+        _weightSelector(this.setFeeNoRevert.selector, 1);
+        _weightSelector(this.setPrice.selector, 5);
+        _weightSelector(this.mine.selector, 100);
```

Comment:
This needs to be before the following `super.setUp()` because the array `selectors` is registered here:
https://github.com/morpho-org/morpho-blue/blob/d4d5e5649c5358a51978a6c913905217b07b4514/test/forge/InvariantTest.sol#L21